src/audio/base_fw.c

```diff
@@ -461,7 +466,9 @@ static int basefw_get_large_config(struct comp_dev *dev,
 	/* TODO: add more support */
 	case IPC4_DSP_RESOURCE_STATE:
 	case IPC4_NOTIFICATION_MASK:
+		break;
```

Comment:
Why is this not even printing a warning?

Reply:
You seem to be splitting the "TODO", unimplemented section in the middle. This means the second part is losing its "TODO" comment and silently doing nothing with zero comment and zero warning. So now it looks even less intentional and even more like a bug.

Reply:
move out the IPC4_MODULES_INFO_GET case from the TODO list.

Reply:
done, currently all implemented cases are before the TODO comment